src/fft.jl

```diff
@@ -60,7 +60,7 @@ Perform a real valued iFFT; see [`ifft`](@ref).
 """
 function irfft(basis::PlaneWaveBasis{T}, f_fourier::AbstractArray; check=Val(true)) where {T}
     f_real = ifft(basis, f_fourier)
-    (check == Val(true)) && @assert norm(imag(f_real)) < sqrt(eps(T))
+    (check == Val(true)) && @assert norm(imag(f_real)) < max(1e-12, sqrt(eps(T)))
```

Comment:
I hate these checks so much. Let's just kill them all...

Reply:
Not sure. They do help during development for sure.